src/Microsoft.ML.Data/Transforms/SelectColumnsTransform.cs

```diff
@@ -0,0 +1,632 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+[assembly: LoadableClass(SelectColumnsTransform.Summary, typeof(IDataTransform), typeof(SelectColumnsTransform),
+                typeof(SelectColumnsTransform.Arguments), typeof(SignatureDataTransform),
+                SelectColumnsTransform.UserName, "SelectColumns", "SelectColumnsTransform", SelectColumnsTransform.ShortName, DocName = "transform/SelectTransforms.md")]
+
+[assembly: LoadableClass(SelectColumnsTransform.Summary, typeof(IDataView), typeof(SelectColumnsTransform), null, typeof(SignatureLoadDataTransform),
+                            SelectColumnsTransform.UserName, SelectColumnsTransform.LoaderSignature)]
+
+[assembly: LoadableClass(SelectColumnsTransform.Summary, typeof(SelectColumnsTransform), null, typeof(SignatureLoadModel),
+                            SelectColumnsTransform.UserName, SelectColumnsTransform.LoaderSignature)]
+
+// Back-compat to handle loading of the Drop and Keep Transformer
+[assembly: LoadableClass("", typeof(IDataView), typeof(SelectColumnsTransform), null, typeof(SignatureLoadDataTransform),
+    "", SelectColumnsTransform.DropLoaderSignature)]
+
+// Back-compat to handle loading of the Choose Columns Transformer
+[assembly: LoadableClass("", typeof(IDataView), typeof(SelectColumnsTransform), null, typeof(SignatureLoadDataTransform),
+    "", SelectColumnsTransform.ChooseLoaderSignature, SelectColumnsTransform.ChooseLoaderSignatureOld)]
+
+namespace Microsoft.ML.Transforms
+{
+    /// <summary>
```

Comment:
>summary [](start = 9, length = 7)

Please re-word this comment. It should not mention that this is an estimator class, and the 2nd sentence looked off. Maybe the clarification about 'it may or may not keep hidden columns' don't really belong to the summary comment